### PR TITLE
Fix message when we have no internet connection

### PR DESCRIPTION
**Pull request by maras93**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -62,6 +62,7 @@
     <string name="empty_deck">Pusta talia</string>
     <string name="no_decks">Brak talii</string>
     <string name="no_decks_query">Brak talii o podanej nazwie</string>
+    <string name="decks_download_error">Błąd pobierania talii</string>
```
Comment (lkopocinski) [inline app/src/main/res/values/strings.xml]: "decks_download_error" and "download_decks_error" have the same message. Is it necessary?

File: app/src/main/java/com/blstream/studybox/data_provider/DataHelper.java [hunk 2]
```diff
@@ -18,6 +21,13 @@
 public class DataHelper implements DataProvider {
 
     private List<Deck> currentDecks;
+    private Resources resources;
```
Comment (lkopocinski) [inline app/src/main/java/com/blstream/studybox/data_provider/DataHelper.java]: Makeing Resources object from android.content.res.Resources class in DataHelper it makes DataHelper dependend on platform. It should be better to use some interface to hide this dependency.

"In theory the Presenter and business logic could be plain old java code, which could be shared with a desktop application or any other java application." http://hannesdorfmann.com/mosby/mvp/

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (maras93) [inline app/src/main/res/values/strings.xml]: "download_decks_error" was deleted

File: app/src/main/java/com/blstream/studybox/data_provider/DataHelper.java (hunk 2 above)
Comment (maras93) [inline app/src/main/java/com/blstream/studybox/data_provider/DataHelper.java]: I aggree with that but in this case I prefered this solution.

**Pull request merged**